README.rdoc

```diff
@@ -9,13 +9,16 @@ It works with ANY class, however, you get a few extra features when you're using
 
   gem install attr_encrypted
 
+Or add to your Gemfile:
+
+  gem 'attr_encrypted', '~> 1.3', '>= 1.3.4'
```

Comment:
I suggest you drop the version constraint. It won't make sense soon and will always changing.